core/src/main/scala/fs2/gzip/AsyncByteArrayInputStream.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2014–2018 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fs2.gzip
+
+import cats.effect.Sync
+
+import scala.collection.mutable
+
+import scala.{math, Array, Boolean, Byte, Int, List, Unit}
+// import scala.{Predef, StringContext}, Predef._
```

Comment:
I honestly hate -Yno-imports because of stuff like this. Println debugging is a massive pain when you need to constantly reimport predef. So I just left the line commented out in every file I debugged so I wouldn’t have to re-type it. Can remove if you prefer.

Reply:
Meh, I called it out assuming it was unintentional, I'm fine with it to make debugging easier.